services/lgtm/lgtm.tester.js

```diff
@@ -67,7 +67,7 @@ t.create('alerts: invalid json')
       .get('/api/v0.1/project/g/apache/cloudstack/details')
       .reply(invalidJSON)
```

Comment:
You can get rid of any tests that use `.reply(invalidJSON)` - once we migrate away from the legacy format, this stuff is handled by the base classes and has its own tests